apps/artwork/components/actions/bootstrap.coffee

```diff
@@ -7,7 +7,7 @@ module.exports = (sd, { artwork }) ->
       _id: artwork._id
 
     share:
-      media: artwork.images[0].url
+      media: if artwork.images[0]?.url then artwork.images[0].url else null
```

Comment:
#minor maybe metaphysics should have a `has_image` or, if the `is_`-prefix is desired, `is_with_image` that wraps up the check.